module-check/src/test/clojure/clojure/core/typed/test/template.clj

```diff
@@ -0,0 +1,23 @@
+(ns clojure.core.typed.test.template
+  (:require 	[clojure.core.typed :as t]
+                [clojure.template :as template] 
+		[clojure.xml :as xml]
+		[clojure.zip :as zip]
+		[clojure.test :refer :all]                
+		[clojure.core.typed.test.test-utils :refer :all]))
+
+(deftest applyTemplate-test
+  (is-tc-e   (clojure.template/apply-template '[a b c d e] '[d a b e c e b a d] '(1 2 3 4 5))t/Any) 
```

Comment:
Every `is-tc-e` expands to a `do` with a `ns` form.

```clojure
(is-tc-e   (apply-template '[a b c d e] '[d a b e c e b a d] '(1 2 3 4 5))
              :requires [[clojure.template :refer [apply-template]]])
```

expands to something like 

```clojure
(is-tc-e  (do (ns temp123213 (:require [clojure.template :refer [apply-template]]))
                  (apply-template '[a b c d e] '[d a b e c e b a d] '(1 2 3 4 5)))
```

This helps you not have to use a clojure.template prefix on everything.